logger/README.md

```diff
@@ -0,0 +1,116 @@
+## Log usage
+
+The samples demonstrate how to configure dubbo-go logger using lumberjack
+
+### Configuration
+
+Output file settings
+
+```yaml
+lumberjackConfig:
+  # The name of the log file
+  filename: "logs.log"
+  # The maximum size of each log file length, the default is 100M
```

Comment:
The maximum size of each log file length, its unit is MiB. The default value is 100MiB.